### DISCUSSION TITLE
How to use with electron and express

### DISCUSSION THREAD

**Discussion by M-Barari** [prisma/prisma#9335] [Q&A]
Hi, I made an app in nuxt.js + electron.js . then added `prisma` to `nuxt` and used `express` in serverMiddleware to handle my endpoints started `api/` and send them to prisma CRUD . It works just fine in dev mode, but when building `.exe` file , there is no serverMiddleware to handle those endpoints. can anyone help me??

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
